hal/src/boron/pinmap_hal.c

```diff
@@ -123,6 +123,58 @@ const uint8_t NRF_PIN_LOOKUP_TABLE[48] = {
 
 #endif // PLATFORM_ID == PLATFORM_BSOM
 
+#if PLATFORM_ID == PLATFORM_ESOMX
+
+static Hal_Pin_Info s_pin_map[TOTAL_PINS] = {
```

Comment:
This pin map looks not match our Evo board schematic. Maybe you will change it later. Please reference: [evo schematic](https://github.com/particle-iot/Evo/tree/main/Eagle/v0.02)